specutils/utils/__init__.py

```diff
@@ -1 +1 @@
-
+from .spectral_region import SpectralRegion
```

Comment:
I would say `SpectralRegion` belongs in `spectrum`, *not* in utils - this is a key concept for interfacing with spectra, after all.  And it should be importable directly as `from specutils import SpectralRegion` since it's a basic user interaction.